tutorials/distributed-keras-on-cmle/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Distributed Keras on Google Cloud Machine Learning Engine
+description: Learn how to use distributed Keras on Cloud Machine Learning Engine.
+author: corrieelston
+tags: Data Science, Cloud Machine Learning Engine, Keras, Python
+date_published: 2017-09-28
+---
+
+# Distributed Keras on Google Cloud Machine Learning Engine (CMLE)
+
+This solution connects the dots between a number of technologies with the end result of a production-grade model.
+
+First the technologies:
+
+* [TensorFlow](https://www.tensorflow.org/) (including [tf.contrib.learn](https://www.tensorflow.org/get_started/tflearn)) - TensorFlow is the most popular deep learning library available.  It is functional and performant, but equally importantly, has a vibrant community of users producing educational and reference materials to accelerate your progress towards adding value to your business.
+* [Keras](https://keras.io/) - Keras is a high-level, Python API focused on enabling fast experimentation, with the motivation that moving from idea to result with the least possible delay is the key to good research (and machine learning in general).  It is being integrated into TensorFlow (currently as tf.contrib.keras) but is also available standalone and can run on TensorFlow, CNTK or Theano.
+* [Google Cloud Machine Learning Engine (CMLE)](https://cloud.google.com/ml-engine/) - CMLE is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.  It facilitates two activities: One, decreasing training time by allowing you train models using large numbers of CPUs, GPUs and TPUs, and two,
+allowing you to train models in parallel.  Both of these activities further enable you to iterate faster with less overhead.
+
+## Prerequisites
+
+* Run through the [Cloud Datalab quickstart](https://cloud.google.com/datalab/docs/quickstarts):
+  * Create a project on Google Cloud Platform.
+  * Ensure Compute Engine (GCE) and CMLE APIs are enabled.
+  * Create an instance of Cloud Datalab.
+* Create a new notebook within Datalab and execute the following from a code cell, `!gsutil cp gs://gcp-community/tutorials/distributed-keras-on-cmle/*.ipynb .`
+* Open `1-start-here.ipynb` and enjoy.
```

Comment:
The high-level bullets here are sequential instructions, and should use an ordered list. Also, these are instructions for the tutorial, not prerequisites. Maybe the prerequisite is that you have a working instance of Cloud Datalab [with a link to the quickstart provided]? Then you could describe how to get started in its own section--"How to use this tutorial" or some such. It would be good to introduce the methodology of using several notebooks that build on each other.